index.d.ts

```diff
@@ -14,14 +24,27 @@ const result = await runAppleScript('return "unicorn"');
 console.log(result);
 //=> 'unicorn'
 ```
+
+@example
+```
+import {runAppleScript} from 'run-applescript';
+
+const options = {
+    humanReadableOutput: false
+}
+const result = await runAppleScript('return "unicorn"', options);
+
+console.log(result);
+//=> '"unicorn"'
+```
```

Comment:
The example should be on the docs for the `humanReadableOutput` option. Line 8.

Reply:
Should I keep the example in both the functions and `humanReadableOutput` docs?

Reply:
No, only in `humanReadableOutput`.

Reply:
Done in cf16b68.